src/test/regress/sql/matview.sql

```diff
@@ -187,6 +187,11 @@ SELECT * FROM mvtest_mv_v_3;
 SELECT * FROM mvtest_mv_v_4;
 DROP TABLE mvtest_v CASCADE;
 
+-- Check that CREATE IF NOT EXISTS accept DISTRIBUTED BY
+CREATE MATERIALIZED VIEW IF NOT EXISTS mv_ine_distr (a, b) AS
+  SELECT generate_series(1, 10) a, generate_series(1, 10) b  DISTRIBUTED BY (b);
+DROP MATERIALIZED VIEW mv_ine_distr;
```

Comment:
`\d+ mv_ine_distr ` to display dist key info.

Reply:
even though the main job of these test in to check that DISTRIBUTED BY and IF NOT EXISTS are accepted together, not the DISTRIBUTED BY value?
Well, maybe this is good for sake of sameness test set as whole. 
Pushed as requested